Casks/c/crystalmaker.rb

```diff
@@ -9,7 +9,10 @@
 
   livecheck do
     url "https://crystalmaker.com/support/updates/VersionData-CMM.xml"
-    regex(/<version[^>]+?number=["']?v?(\d+(?:\.\d+)+)["']?[^>]*?>/i)
+    regex(/\b(\d+\.\d+\.\d+)\b/)
+    strategy :xml do |xml, regex|
+      xml.get_elements("versiondata/versionlist/version").map { |item| item.attributes["number"][regex, 1] }
```

Comment:
```suggestion
    strategy :xml do |xml|
      xml.get_elements("//version").map { |item| item.attributes["number"] }
```

Same as https://github.com/Homebrew/homebrew-cask/pull/172222#discussion_r1581568335.